plugins/ftdidmx/FtdiDmxThread.cpp

```diff
@@ -164,9 +496,23 @@ void *FtdiDmxThread::Run() {
 void FtdiDmxThread::CheckTimeGranularity() {
   TimeStamp ts1, ts2;
   Clock clock;
+  timespec req, rem;
```

Comment:
Can you look at integrating this with http://docs.openlighting.org/ola/doc/latest/classola_1_1_time_interval.html please.